auror_core/v2/__init__.py

```diff
@@ -0,0 +1,12 @@
+from auror_core.v2.job import Command
+
+
+class JobType:
+    COMMAND = Command
+
+    @staticmethod
+    def get_job_type_class(job_type):
+        _job_type = getattr(JobType, job_type.upper(), None)
+        if not _job_type:
+            raise ValueError('Invalid job type: {}'.format(job_type))
```

Comment:
Fallback to type "Command" if you don't find JobType class because all JobTypes extends from "Command" but with new config params

Reply:
Nice, I saw others Job Types on `azkaban` project but at this project we just can find(in version 2) `Command` Job type.